optimize_versioned_sidebars/version-3.7.0-sidebars.json

```diff
@@ -3,50 +3,50 @@
     {
       "type": "link",
       "label": "Overview Components",
-      "href": "/docs/1.3/components/"
```

Comment:
This is not fun, but could be automated with `sed`. These URLs are all cross-docs-instance. Since I removed 1.3 from the path of all pages, I needed to remove it in the sidebars too.